hot-reload-orchestration/src/main/kotlin/org/jetbrains/compose/reload/orchestration/OrchestrationMessage.kt

```diff
@@ -53,6 +53,7 @@ public sealed class OrchestrationMessage : Serializable {
      *
      * Note: This message can be sent multiple times!
      */
+    @Deprecated("This message is not required anymore")
```

Comment:
Do we need to keep this class if it is not used anymore? If yes, maybe we should add a small comment on wty it's still here and why its deprecated?